guiEditor/src/diagram/guiNode.ts

```diff
@@ -181,18 +185,28 @@ export class GUINode {
         }
     }
 
-    public addGui(childNode: GUINode) {
-        if (!this._isContainer) return;
+    public addChildGui(childNode: GUINode) {
+        if (!this.isContainer) return;
+        childNode.parent = this;

```

Comment:
is that needed? can't you get this information from the scene itself, after setting the element's parent (or container)?

Reply:
See below comments.